shell/app/atom_content_client.h

```diff
@@ -24,6 +24,7 @@ class AtomContentClient : public content::ContentClient {
   base::StringPiece GetDataResource(int resource_id, ui::ScaleFactor) override;
   gfx::Image& GetNativeImageNamed(int resource_id) override;
   base::RefCountedMemory* GetDataResourceBytes(int resource_id) override;
+  base::FilePath ChromeContentBrowserClient::GetFontLookupTableCacheDir();
```

Comment:
Why was this added ? Can you link the CL related to this in the commit. Also the namespace is incorrect.

Reply:
it was added as an attempt to address a failure caused by this: https://chromium-review.googlesource.com/c/chromium/src/+/1753006